clients/admin-ui/src/types/api/models/SystemType.ts

```diff
@@ -9,4 +9,5 @@ export enum SystemType {
   SAAS = "saas",
   DATABASE = "database",
   MANUAL = "manual",
+  EMAIL = "email",
```

Comment:
for future reference, these are autogenerated so you shouldn't need to modify them manually. though we only _just_ [got our autogenerated types up to date](https://github.com/ethyca/fides/pull/2567) and if you had tried to autogenerate when you made this PR you would have had a very bad time 😅

Reply:
ah ok!